core/modules/layout/plugins/context/entity_layout_context_handler.inc

```diff
@@ -78,12 +78,11 @@ class EntityLayoutContext extends LayoutContext {
    * Submit handler for the settings form for this context item.
    */
   function formSubmit($form, &$form_state) {
-    $excluded = backdrop_map_assoc(array('cancel', 'submit', 'form_build_id', 'form_token', 'form_id', 'op', 'add', 'context'));
+    $excluded = backdrop_map_assoc(array('cancel', 'submit', 'form_build_id', 'form_token', 'form_id', 'op', 'add', 'context', 'load_context_nojs'));
     $settings = array_diff_key($form_state['values'], $excluded);
     // This produces config keys for contexts in the format "node1" or "user3".
     $this->name = $this->plugin . $settings['id'];
-
-    $this->settings = array_merge($this->settings, $settings);
+    $this->settings = $settings;
```

Comment:
I removed the merging here so that only the values sent in the form are saved. Otherwise any accidental value (such as the "load_context_nojs" value) will be stuck in there even after resaving the form.